metricbeat/module/vsphere/host/host.go

```diff
@@ -29,7 +29,7 @@ type MetricSet struct {
 }
 
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
-	logp.Experimental("The vsphere host metricset is experimental")
+	logp.Warn("EXPERIMENTAL: The vsphere host metricset is experimental")
```

Comment:
Sorry. I tried to make the same as here so I change this warning.
https://github.com/elastic/beats/pull/4220/files